content/courses/onchain-development/local-setup.md

```diff
@@ -81,32 +108,47 @@ anchor test
 ```
 
 **The `anchor test` command should complete with no errors or warnings**.
-However you may encounter issues, and we'll fix them below:
 
-##### `package `solana-program v1.18.12` cannot be built because it requires rustc 1.75.0 or newer` error
+**However you may encounter issues, and we'll fix them below:**
+
+#### `package `solana-program
+
+v1.18.12` cannot be built because it requires rustc 1.75.0 or newer` error
 
+This error is due to incompatible versions of `solana-program` and `solana-cli`.
 Run `cargo add solana-program@"=1.18.x"`, where `x` matches your version of
 `solana-cli`. Then re-run `anchor test`.
 
-##### `Error: Unable to read keypair file`
+#### `Error: Unable to read keypair file`
 
 Add a keypair to `.config/solana/id.json`. You can either copy a keypair from an
 `.env` file (just the array of numbers) into a file or use the command
 `solana-keygen new --no-bip39-passphrase` to create a new keypair file. Then
 re-run `anchor test`.
 
-##### `unused variable: 'ctx'` warning
+#### error: no such command: `build-sbf`
+
```

Comment:
It's worth adding an explanation for why the error is occurring. Eg "after installing Solana CLI, you'll need to ensure it's executables are in your PATH"...